xml/System.Diagnostics.Tracing/EventWrittenEventArgs.xml

```diff
@@ -29,7 +29,33 @@
   <Interfaces />
   <Docs>
     <summary>Provides data for the <see cref="M:System.Diagnostics.Tracing.EventListener.OnEventWritten(System.Diagnostics.Tracing.EventWrittenEventArgs)" /> callback.</summary>
-    <remarks>To be added.</remarks>
+    <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+
+Whenever an event is dispatched to an <xref:System.Diagnostics.Tracing.EventListener>, the <xref:System.Diagnostics.Tracing.EventListener.OnEventWritten&2A?displayProperty=nameWithType> callback method is invoked. It is passed an `EventWrittenEventArgs` instance that contains information associated with the event. 
+
+The following sections contain additional information about individual `EventWrittenEventArgs` properties.
+
+### ActivityId property
```

Comment:
You can remove this section on the ActivityId property.   It is subsumed by the suggestion above (which is that all fields are only valid during the callback.    I would not worry about ActivityId changing during the callback, it is very unlikely to be relevant, but I leave that to you.